Description: Instruct man to preserve formatting characters in its output
 rather than filtering them through col. Works as of man-db 2.5.0, but is
 harmless with older versions.
Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=325699
Ubuntu: https://bugs.launchpad.net/ubuntu/+source/w3m/+bug/353900

Index: b/scripts/w3mman/w3mman2html.cgi.in
===================================================================
--- a/scripts/w3mman/w3mman2html.cgi.in
+++ b/scripts/w3mman/w3mman2html.cgi.in
@@ -76,7 +76,7 @@
   if (! ($file =~ /^\//)) {
     $file = $query{"pwd"} . '/' . $file;
   }
-  open(F, "$MAN -l $file 2> /dev/null |");
+  open(F, "MAN_KEEP_FORMATTING=1 $MAN -l $file 2> /dev/null |");
 } else {
   $man = $query{"man"};
   if ($man =~ s/\((\w+)\)$//) {
@@ -92,7 +92,7 @@
 
   $section =~ s:([^-\w\200-\377.,])::g;
   $man =~ s:([^-\w\200-\377.,])::g;
-  open(F, "$MAN $section $man 2> /dev/null |");
+  open(F, "MAN_KEEP_FORMATTING=1 $MAN $section $man 2> /dev/null |");
 }
 $ok = 0;
 undef $header;
